src/equations/NavierStokes_utils.jl

```diff
@@ -6,6 +6,8 @@ module NavierStokes
 
 using IncompressibleNavierStokes: IncompressibleNavierStokes as INS
 using Lux: Lux
+using Zygote
+using ChainRulesCore
```

Comment:
@SCiarella can you explain the need for these two import statements?
If they are needed, I suggest to explicitly import them as:
```
using Zygote : Zygote
```
and qualify **ALL** usages of the pakage.